dashboard/app/controllers/api/v1/user_school_infos_controller.rb

```diff
@@ -13,10 +13,21 @@ def update_last_confirmation_date
   def update
     return unless school_info_params[:school_id].present? || school_info_params[:country].present?
 
+    new_school_info_params =
+      if school_info_params[:full_address]&.blank?
+        school_info_params.except(:full_address)
+      else
+        school_info_params
+      end
```

Comment:
Just a suggestion: You can express this more compactly with [`except!`](https://apidock.com/rails/Hash/except%21) or [`delete`](https://apidock.com/ruby/Hash/delete), mutating the hash in-place.

```rb
school_info_params.delete(:full_address) if school_info_params[:full_address]&.blank?
```

EDIT: But noticed this suggestion does not introduce the new temporary variable; interested in how attached you are to that.

Reply:
@islemaster Thanks for the suggestion.  I was tinkering with different options to make the code more compact.  One initial solution I considered was to add a method that returns the value for school info was to create a method that would return the school info params.  This works!

Reply:
@islemaster I will open another PR for the method refactor